pkgdb/tests/lock-flake-installable.cc

```diff
@@ -372,6 +384,7 @@ main( int argc, char * argv[] )
   RUN_TEST( version, state, system );
   RUN_TEST( broken, state, system );
   RUN_TEST( unfree, state, system );
+  RUN_TEST( priority, state, system );
```

Comment:
blocking: buildenv needs to respect priority by changing `pkg.priority  = resolver::DEFAULT_PRIORITY;` in `buildenv-lockfile.cc`. And then we probably need a test case in `buildenv-lockfile.cc` checking that priority correctly gets passed through when loading `BuildenvLockfile` from a lockfile with flakes

Reply:
🤦‍♂️